src/ray/gcs/gcs_client/accessor.h

```diff
@@ -459,7 +459,7 @@ class NodeResourceInfoAccessor {
   /// server.
   virtual void AsyncResubscribe();
 
-  /// Report resource usage of a node to GCS asynchronously.
+  /// Report resource usage of a node to GCS asynchronously. Only used in tests.
```

Comment:
Remove?

Reply:
This is a cleanup which I think should be convenient to include here. Added to the PR description.